kubernetes/omsagent.yaml

```diff
@@ -1,728 +1,731 @@
+            - mountPath: /etc/config/settings/adx
+              name: omsagent-adx-secret
+              readOnly: true
+          livenessProbe:
+            exec:
+              command:
+                - /bin/bash
+                - -c
+                - /opt/livenessprobe.sh
+            initialDelaySeconds: 60
+            periodSeconds: 60
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - labelSelector:
+                matchExpressions:
+                  - key: beta.kubernetes.io/os
+                    operator: In
+                    values:
+                    - linux
+                  - key: type
+                    operator: NotIn
+                    values:
+                      - virtual-kubelet
+      # Tolerate a NoSchedule taint on master that ACS Engine sets.
+      tolerations:
+        - operator: "Exists"
+          effect: "NoSchedule"
+        - operator: "Exists"
+          effect: "NoExecute"
+        - operator: "Exists"
+          effect: "PreferNoSchedule"
+      volumes:
+        - name: host-root
+          hostPath:
+            path: /
+        - name: docker-sock
+          hostPath:
+            path: /var/run
+        - name: container-hostname
+          hostPath:
+            path: /etc/hostname
+        - name: host-log
+          hostPath:
+            path: /var/log
+        - name: containerlog-path
+          hostPath:
+            path: /var/lib/docker/containers
+        - name: azure-json-path
+          hostPath:
+            path: /etc/kubernetes
+        - name: omsagent-secret
+          secret:
+            secretName: omsagent-secret
+        - name: settings-vol-config
+          configMap:
+            name: container-azm-ms-agentconfig
+            optional: true
+        - name: omsagent-adx-secret
+          secret:
+            secretName: omsagent-adx-secret
+            optional: true
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: omsagent-rs
+  namespace: kube-system
+  labels:
+    component: oms-agent
+    tier: node
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      rsName: "omsagent-rs"
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        rsName: "omsagent-rs"
+      annotations:
+        agentVersion: "1.10.0.1"
+        dockerProviderVersion: "10.0.0-1"
+        schema-versions: "v1"
+    spec:
+      serviceAccountName: omsagent
+      containers:
+        - name: omsagent
+          image: "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod05262020"
+          imagePullPolicy: IfNotPresent
+          resources:
+            limits:
+              cpu: 1
+              memory: 750Mi
+            requests:
+              cpu: 150m
+              memory: 250Mi
+          env:
+            # azure devops pipeline uses AKS_RESOURCE_ID and AKS_REGION hence ensure to uncomment these
```

Comment:
why change default from non-aks to AKS ?

Reply:
> why change default from non-aks to AKS ?

> pipelines are replacing these place-holders to deploy onto ci_dev and ci_prod clusters. do you see any issue  changing this?

Reply:
> > why change default from non-aks to AKS ?
> 
> > pipelines are replacing these place-holders to deploy onto ci_dev and ci_prod clusters. do you see any issue  changing this? for non-AKS, clusters, release pipelines will uses the HELM chart which I am planning to do next PR.

Reply:
no, not an issue. should be ok.